deploy/20_ocm-agent.Role.yaml

```diff
@@ -19,3 +19,13 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ocmagent.managed.openshift.io
+    resources:
+      - managednotifications
+    verbs:
```

Comment:
should we give all the permissions to the role since the CR is under the ocmagent api group?

Reply:
I couldn't really see any reason why the `ocm-agent` SA would need to delete the ManagedNotification CR.